lib/ansible/modules/apt.py

```diff
@@ -205,22 +205,22 @@
         platforms: debian
 notes:
    - Three of the upgrade modes (V(full), V(safe) and its alias V(true)) required C(aptitude) up to 2.3, since 2.4 C(apt-get) is used as a fall-back.
-   - In most cases, packages installed with apt will start newly installed services by default. Most distributions have mechanisms to avoid this.
+   - In most cases, packages installed with C(apt) will start newly installed services by default. Most distributions have mechanisms to avoid this.
```

Comment:
```suggestion
   - In most cases, packages installed with I(apt) will start newly installed services by default. Most distributions have mechanisms to avoid this.
```